subsys/shell/shell_uart.c

```diff
@@ -273,6 +275,10 @@ static int enable_shell_uart(struct device *arg)
 	u32_t level =
 		(CONFIG_SHELL_BACKEND_SERIAL_LOG_LEVEL > LOG_LEVEL_DBG) ?
 		CONFIG_LOG_MAX_LEVEL : CONFIG_SHELL_BACKEND_SERIAL_LOG_LEVEL;
+	
+	if (IS_ENABLED(CONFIG_UART_SHELL_USE_CDC_ACM)) {
+		usb_enable(NULL);
```

Comment:
As described for #27071 this hijacks a call that's supposed to be available to the application to register for USB event status changes.